### PR TITLE
chore(deps): Integrate dependabot suggested changes.

### PR DESCRIPTION
**Pull request by drstrangelooker**

### CODE REVIEW

**Review by joeldodge79 — DISMISSED** [review]
sweet, thanks!

**Comment by drstrangelooker** [comment]
replaced by https://github.com/looker-open-source/sdk-codegen/pull/991

**Review by joeldodge79 — APPROVED** [review]

File: package.json
Comment (joeldodge79) [inline package.json:265]: what does this do?

**Review by joeldodge79 — COMMENTED** [review]
Gonna add @bryans99 because this just got beyond my currently available JS/TS skills

File: yarn.lock
Comment (bryans99) [inline yarn.lock:10876]: node-forge 0.10.0 deleted

File: yarn.lock
Comment (bryans99) [inline yarn.lock:10919]: And the resolution of node-forge 0.10.0 is now version 1.0.0 (line below)

File: package.json
Comment (bryans99) [reply package.json:265]: The resolutions section overrides the dependencies pulled into yarn.lock.

Lets look at node-forge

Prior to this, something dragged in a dependency that resolved to node-forge 0.10.0 (selfsigned is one package that uses it). The above is telling yarn (lerna?) to use node-forge 1.0.0 in yarn.lock.

I've added a couple of comments below.

The others are doing the same thing. If the tests pass we should be okay. ideally we would update the top level package that indirectly references it and that would update all of the offending dependencies. Unfortunately its never that easy.

**Comment by joeldodge79** [comment]
Once this PR is merged, does that mean that `yarn.lock` will NOT show up as a modified file after a `yarn install` or `yarn build` - currently it does and that's borking the package publishing step. So I'd like to hold off on the [22.2 release](https://github.com/looker-open-source/sdk-codegen/pull/990#pullrequestreview-887675542) till we resolve that

**Comment by drstrangelooker** [comment]
That should be the case. Yarn.lock should not change.

**Review by drstrangelooker — COMMENTED** [review]
LGTM but I can't approve my own changes. Pulled and reran yarn with no changes to yarn.lock, so it should be ok.

File: examples/access-token-server/package.json
Comment (bryans99) [inline examples/access-token-server/package.json:39]: Is node-fetch needed here?

Comment (drstrangelooker) [reply examples/access-token-server/package.json:39]: yes https://github.com/looker-open-source/sdk-codegen/security/dependabot/21

Comment (bryans99) [reply examples/access-token-server/package.json:39]: Sorry, should have been clearer. node-fetch ^2.6.7 is included as a direct dependency in package.json so I was wondering if this is needed. Will approve anyway

**Review by bryans99 — APPROVED** [review]

**Pull request merged**
